cpp/unit/action.cpp

```diff
@@ -69,10 +69,11 @@ void UnitAction::damage_object(Unit &target, unsigned dmg) {
 	}
 }
 
-void UnitAction::move_to(Unit &target) {
+void UnitAction::move_to(Unit &target, bool use_range) {
```

Comment:
A boolean here is good, but using an enum class would increase readability instead of just having true and false as "visible" parameters. Don't you think?